src/add-image/index.md

```diff
@@ -0,0 +1,21 @@
+# Adding images to my game page
+
+Images may be the most vital element of your game page in terms of catching the attention of people who have already been enticed by your thumbnail, or who are visiting via a direct link to your page. Many people will simply skip over a game with no screenshots.
+
+If your game is a work-in-progress, it's essential to post images in order to get--and keep--people interested in your game. Screenshots in particular are excellent proof of progress.
+
+Images can keep players coming back for more, for instance if they've glimpsed things in the images that they haven't yet seen in the game.
```

Comment:
should we remove this section? it sounds like an intro but in the middle of the section.